cmd/tweed/broker.go

```diff
@@ -117,14 +148,82 @@ func Broker(args []string) {
     prefix  :: @C{%s}
     vault   :: @C{%s}
 
-`, opts.Broker.Config, opts.Broker.Root, opts.Broker.Listen, core.Config.Prefix, core.Config.Vault.Prefix)
+`, cmd.Config, cmd.Root, cmd.Listen, core.Config.Prefix, core.Config.Vault.Prefix)
 
 	fmt.Fprintf(os.Stderr, "waiting for vault to open up for business...\n")
 	core.WaitForVault()
 
-	core.KeepErrors(opts.Broker.KeepErrors)
+	core.KeepErrors(cmd.KeepErrors)
 
 	fmt.Fprintf(os.Stderr, "tweed broker API spinning up...\n")
 	http.Handle("/b/", core.API())
-	http.ListenAndServe(opts.Broker.Listen, nil)
+	http.ListenAndServe(cmd.Listen, nil)
+
+	return nil
+}
+
+func (cmd *BrokerCommand) WireDynamicFlags(commandFlags *flags.Command) {
```

Comment:
The switch to go-flags was motivated by this method. It allows a dynamic number of credential managers to register their configuration flags with the main broker command.